PRDs/20251215-shell-completion/PRD.md

```
@@ -0,0 +1,653 @@
+                    _arguments \
+                        '--help[Show help]' \
+                        '--to[Output file]:file:_files' \
+                        '1:shell:(bash zsh)'
+                    ;;
+            esac
+            ;;
+    esac
+}
+
+_metaschema-cli "$@"
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)